databuilder/extractor/minio_extractor.py

```diff
@@ -1,9 +1,14 @@
-from pyhocon import ConfigTree
-from typing import Any
+from typing import Any, Iterator
+
```

Comment:
Extra whitespace between imports

Reply:
This is actually intentional -- I'm grouping my imports into three groups according to PEP-8:
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

You should put a blank line between each group of imports.
```

https://www.python.org/dev/peps/pep-0008/